manager/utils.go

```diff
@@ -179,7 +178,12 @@ func GenerateEventName(probeType, funcName, UID string, attachPID int) (string,
 
 	if len(eventName) > MaxEventNameLen {
 		// truncate the function name and UID name to reduce the length of the event
-		eventName = safeEventRegexp.ReplaceAllString(fmt.Sprintf("%s_%s_%s_%d", probeType, funcName[0:int(math.Min(10, float64(len(funcName))))], UID[0:int(math.Min(10, float64(len(UID))))], attachPID), "_")
+		attachPIDstr := strconv.Itoa.attachPID
```

Comment:
should this be `strconv.Itoa(attachPID)`?

Reply:
Thanks, I was assuming the CI would detect the compilation errors,  I added an local (prepush hook) test

Reply:
We don't have CI on this repo yet :(